tests/utils.py

```diff
@@ -410,3 +415,4 @@ class Constants:
     NORMAL_TEZOS_ADDRESS = "tz1N4UfQCahHkRShBanv9QP9TnmXNgCaqCyZ"
     STAKENOW_ADDRESS = "tz1g8vkmcde6sWKaG2NN9WKzCkDM6Rziq194"
     BAKEXTZ4ME_ADDRESS = "tz1NRGxXV9h6SdNaZLcgmjuLx3hyy2f8YoGN"
```

Comment:
Testnet address would be the StakeNow Ghostnet baker? and BAKEXTZ4ME is not used anymore? Then it should be removed

Reply:
I didn't change this? The BAKEXTZ4ME address is being used in a number of tests. What would you like me to update it to? using the stakenow address? I think the issue is that the stakenow address has a large amount of delegates and the BAKEXTZ4ME address is a lot smaller?

Reply:
This diff shows you have added TESTNET_ADDRESS, used it in a test which before used the BAKEXTZ4ME_ADDRESS .